### DISCUSSION TITLE
Family/Twin information

### DISCUSSION THREAD

**Discussion by sanaashrafali** [now-i-know-my-abcd/docs#24] [Stats & Code Help]
Does anyone have example code where they selected only one person from each family or only one twin?

I think the following works but am not super confident if folks want to compare code or check/share

<img width="314" alt="Screen Shot 2022-08-03 at 10 13 51 AM" src="https://user-images.githubusercontent.com/103009041/182668988-d840a521-1389-4553-9a9d-ec8da4bce67a.png">

https://www.datasciencemadesimple.com/select-random-samples-r-dplyr-sample_n-sample_frac/

**Comment by jflournoy** [comment]
Here's a reproducible code example, and I think it confirms that your method works!

```
library(tidyr)
library(dplyr)
d <- data.frame(id = 1:100, famid = sample(1:50, 100, replace = TRUE))

hist(table(d$famid))

d_sib_select <- d %>% 
  group_by(famid) %>%
  slice_sample(n = 1)

hist(table(d_sib_select$famid))
```
